en/config_vtol/vtol_without_airspeed_sensor.md

```diff
@@ -62,7 +62,7 @@ Set the minimum front transition time ([VT_TRANS_MIN_TM](../advanced_config/para
 
 ### Optional Recommended Parameters
 
-Because the risk of stalling is real, it is recommended to set the 'fixed wing minimum altitude' aka 'QuadChute' ([VT_FW_MIN_ALT](../advanced_config/parameter_reference.md#VT_FW_MIN_ALT)). 
+Because the risk of stalling is real, it is recommended to set the 'fixed wing minimum altitude' aka 'Quad-chute' ([VT_FW_MIN_ALT](../advanced_config/parameter_reference.md#VT_FW_MIN_ALT)). 

```

Comment:
@sfuhrer Does this sentence make sense - specifically it looks like it is missing a word:
```suggestion
Because the risk of stalling is real, it is recommended to set the 'fixed wing minimum altitude' aka 'quad-chute' threshold ([VT_FW_MIN_ALT](../advanced_config/parameter_reference.md#VT_FW_MIN_ALT)). 

```

Reply:
I would actually not refer to a specific quad-chute threshold here, just add a link to the quad-chute sections where all the different thresholds are defined.